Pyrado/scripts/training/qq-su_sprl-ppo.py

```diff
@@ -45,6 +45,7 @@
 from pyrado.policies.feed_back.fnn import FNNPolicy
```

Comment:
Again, only one qq-su_spdr-ppo.py file. Maybe work with args from the command line to run different flavors.